### ISSUE TITLE
Create Context Program?

### ISSUE THREAD

**Issue by JamesEConnor** [JamesEConnor/NanoGeno2017#5]
What do you think about working on a program that generates random sentences in context to another sentence?

**Comment by JamesEConnor** [comment]
Rather, another script inside the project.

**Comment by skylar-kantor** [comment]
And how would we go about making this “understand” meaning?

**Comment by JamesEConnor** [comment]
Well, I was thinking about implementing a dictionary and thesaurus for it
to find similar words, and pass them into the sentence creator.

**Comment by skylar-kantor** [comment]
Something like this?
https://pypi.python.org/pypi/py-thesaurus/0.95

**Comment by JamesEConnor** [comment]
Yeah, but I already found an API for download, so we can let the program run offline.

**Comment by skylar-kantor** [comment]
Okay. That would be offline as well. Just have to install the library
